.github/workflows/pull_request.yaml

```diff
@@ -0,0 +1,35 @@
+name: iOS Pull Request
+
+on:
+  pull_request:
```

Comment:
Do we also want CI to be triggered when a branch push occurs? Sometimes useful when testing changes vs having to open a PR

Reply:
Yeah, I will make more workflows for those and one that can be triggered manually